test/test_events.py

```diff
@@ -221,23 +225,35 @@ def test_UQCS_events_full(uqcsbot: MockUQCSBot):
     uqcsbot.post_message(TEST_CHANNEL_ID, "!events full")
     messages = uqcsbot.test_messages.get(TEST_CHANNEL_ID, [])
     assert len(messages) == 2
-    expected = ("List of *all* upcoming events\r\n"
-                "*AUG 2 0:00 - AUG 5 23:59* - `CodeNetwork Hackathon` - _River City Labs_\r\n"
-                "*AUG 2 18:00 - 20:00* - `Dr Corey Shou` - _TBC_\r\n"
-                "*AUG 6 18:00 - 20:00* - `vim &amp; tmux - Neil Ashford` - _Hawken 50-T103_\r\n"
-                "*AUG 13 18:00 - 20:00* - `Robogals x UQ Robotics x UQCS Social Event`"
-                + " - _Hawken 50-C207_\r\n"
-                "*AUG 20 18:00 - 20:00* - `How to Hackathon` - _Hawken 50-T103_\r\n"
-                "*AUG 22 18:00 - 20:00* - `C++ Talk - Dr Joel Fenwick` - _Hawken 50-T103_\r\n"
-                "*AUG 23 18:00 - AUG 25 20:00* - `UQCS Hackathon` - _AEB 49-301_\r\n"
-                "*AUG 29 18:00 - 20:00* - `Six Key Things - Peter Laurie` - _Hawken 50-T103_\r\n"
-                "*SEP 5 18:00 - 20:00* - `3D printing: The Future for DIY - Stephanie Piper`"
-                + " - _Hawken 50-T103_\r\n"
-                "*SEP 19 18:00 - 20:00* - `Intro to Firebase - Max Bo` - _Hawken 50-T103_\r\n"
-                "*OCT 8 18:00 - 20:00* - `UQCS AGM` - _TBC_\r\n"
-                "*OCT 17 17:00 - 21:00* - `Games Night` - _TBC_\r\n"
-                "*OCT 29 0:00 - OCT 30 23:59* - `Study Session` - _ITLC: GP South 78-217_\r\n"
-                "*OCT 31 0:00 - NOV 1 23:59* - `Study Session` - _ITLC: GP South 78-217_")
+    expected = (
+        "List of *all* upcoming events\r\n"
+        "*FRI AUG 2 0:00 - MON AUG 5 23:59* - `CodeNetwork Hackathon` - _River City Labs_\r\n"
+        "*FRI AUG 2 18:00 - 20:00* - `Dr Corey Shou` - _TBC_\r\n"
+        "*TUE AUG 6 18:00 - 20:00* - `vim &amp; tmux - Neil Ashford` - "
+        "_Hawken 50-T103_\r\n"
+        "*TUE AUG 13 18:00 - 20:00* - `Robogals x UQ Robotics x UQCS "
+        "Social Event`"
+        + " - _Hawken 50-C207_\r\n"
+        "*TUE AUG 20 18:00 - 20:00* - `How to Hackathon` - _Hawken "
+        "50-T103_\r\n"
```

Comment:
```+ " - _Hawken 50-T103_\r\n"``` and update previous line (keep similar information on the same line when possible).